PlasterTemplate/PlasterTemplates/GitLab/psake.ps1

```diff
@@ -277,8 +277,20 @@ Task Deploy -Depends TestAfterBuild {
     if ($PLASTER_PARAM_DeployDocs -eq 'Yes') {
         '   "`tSetting git repository url"'
         '   # This may need to be changed if your gitlab server does not use HTTPS'
-        '   $GitLabUrl = ''https://oauth2:{0}@<%= $PLASTER_PARAM_GitLabHostname %>/<%= $PLASTER_PARAM_GitLabUserName %>/{1}.git'' -f $ENV:GitLab_PAT, $env:BHProjectName'
-        '   # Get the release version for the commit message'
+        '   $GitLabURL = [uri]$PLASTER_PARAM_GitLabURL'
+        '   if ($GitLabURL.Scheme) {'
+        '       $GitLabProtocol = $GitLabURL.Scheme'
+        '   }'
+        '   else {'
+        '       $GitLabProtocol = ''https'''
+        '   }'
+        '   if ($GitLabURL.Host) {'
+        '       $GitLabHost = $GitLabURL.Host'
+        '   }'
+        '   else {'
+        '       $GitLabHost = $GitLabURL.OriginalString'
+        '   }'
```

Comment:
Sorry for the confusion - these should not be enclosed in quotes. Anything in quotes gets written to the file but in this case we don't want this stuff in the file itself, we need it to be evaluated at run time to populate the variables for the next line.

If you remove the quotes and then decrease the indentation 1 level to line up properly then I think we're good to merge! 🙂